code/modules/antagonists/heretic/knowledge/lock_lore.dm

```diff
@@ -76,10 +76,12 @@
 	if(ismecha(target))
 		var/obj/vehicle/sealed/mecha/mecha = target
 		mecha.dna_lock = null
+		mecha.mecha_flags &= ~ID_LOCK_ON
 		for(var/mob/living/occupant as anything in mecha.occupants)
 			if(isAI(occupant))
 				continue
 			mecha.mob_exit(occupant, randomstep = TRUE)
+			occupant.Paralyze(50)
```

Comment:
```suggestion
			occupant.Paralyze(5 SECONDS)
```

This is translated into decisceoncds, so the value is the same but it is easier to parse the meaning when using this define.